src/views/TipsList.vue

```diff
@@ -200,6 +233,17 @@
           }
         }
 
+        const currencyInstance = new Currency();
+        const getCurrencyRates = currencyInstance.getRates().catch(console.error);
+        const currencyRates = await Promise.resolve(getCurrencyRates);
```

Comment:
why not inline await in previous line?

Reply:
I think the whole currency thins should EITHER be not blocking OR parallel with fetching tips